tests/Unit/NumericalAlgorithms/LinearOperators/Python/Test_PowerMonitors.py

```diff
@@ -36,6 +37,38 @@ def test_power_monitors(self):
 
         np.testing.assert_allclose(np_test_array, np_check_array, 1e-12, 1e-12)
 
+    # Check that the truncation error for a constant unit function is
+    # consistent with numerical floor when we use a high order basis
+    def test_relative_truncation_error(self):
+        num_points_per_dimension = 3
+
+        extent = num_points_per_dimension
+        basis = Basis.Legendre
+        quadrature = Quadrature.GaussLobatto
+        mesh = Mesh2D(extent, basis, quadrature)
+
+        np_vec = np.ones(mesh.number_of_grid_points())
+        test_vec = DataVector(np_vec)
+
+        test_relative_truncation_error_exponent = np.asarray(
+            relative_truncation_error(test_vec, mesh))
+
+        test_relative_truncation_error = np.power(
+            10.0 * np.ones(2), -1.0 * test_relative_truncation_error_exponent)
+
+        expected_truncation_error = 1.0e-16 * np.ones(2)
```

Comment:
You can delete this and use just `0.` below, since you only check to a tolerance of 1e-12 anyway